tests/cases/layout/opacity_in_layout.60

```diff
@@ -14,7 +14,16 @@ export TestCase := Window {
             opacity: 1;
         }
     }
+
+    // issue #915
+    layout3 := HorizontalLayout {
+        for _ in 3 : Rectangle {
+            width: 10px;
+            opacity: 0.5;
+        }
+    }
```

Comment:
Do I understand correctly that  here the Rectangle would become an "optimised" element because only the `width` is set, the opacity is moved into the hidden `Opacity` element?